CHANGELOG.next.asciidoc

```diff
@@ -53,12 +53,13 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 - Fix concurrent map writes when system/process code called from reporter code {pull}32491[32491]
 - Log errors from the Elastic Agent V2 client errors channel. Avoids blocking when error occurs communicating with the Elastic Agent. {pull}34392[34392]
 - Only log publish event messages in trace log level under elastic-agent. {pull}34391[34391]
+- Fix issue where updating a single unit results in other units being turned off. {pull}34504[34504]
```

Comment:
```suggestion
- Fix issue where updating a single Elastic Agent configuration unit results in other units being turned off. {pull}34504[34504]
```

Let's directly mention agent directly since this is the Beats repository.